framework/src/base/Assembly.C

```diff
@@ -145,7 +145,8 @@ Assembly::Assembly(SystemBase & sys, THREAD_ID tid)
   // _coord_msm
   _fe_msm->get_xyz();
 
-  _extra_elem_ids.resize(_mesh.getMesh().n_elem_integers());
+  _extra_elem_ids.resize(_mesh.getMesh().n_elem_integers() + 1);
+  _neighbor_extra_elem_ids.resize(_mesh.getMesh().n_elem_integers() + 1);
```

Comment:
The last extra entry is for subdomain ID, which is technically an element ID. I want to push it into these vectors so that we can switch postprocessings from different element IDs including subdomain ID easily.

Reply:
Sounds fine. Just make sure this is documented clearly somewhere